bin/i18n/translation_status/translation_service.rb

```diff
@@ -0,0 +1,57 @@
+CDO_PROJECT_ROOT = File.expand_path('../../../..', __FILE__).to_s
+DASHBOARD_ROOT = File.join(CDO_PROJECT_ROOT, 'dashboard').to_s
+PEGASUS_ROOT = File.join(CDO_PROJECT_ROOT, 'pegasus').to_s
+LIB_ROOT = File.join(CDO_PROJECT_ROOT, 'lib').to_s
+# Add code-dot-org/lib to the load path.
+$LOAD_PATH.unshift File.expand_path(LIB_ROOT)
+# Add code-dot-org to the load path.
+$LOAD_PATH.unshift File.expand_path(CDO_PROJECT_ROOT)
+require 'i18n'
+require 'i18n/backend/fallbacks'
+require 'deployment'
+require 'active_support'
+require 'cdo/i18n_backend'
+
+# Provides access to all the translations managed by the code-dot-org project.
+class TranslationService
+  def initialize
+    super
+    # Dashboard translations
```

Comment:
Question: are the code below a combination of all i18n-related things we do when loading a Pegasus environment (pegasus/src/env.rb) and starting a Dashboard application (dashboard/config/application.rb)?

Reply:
Yes it is. However, I have removed all of this and opted to use the actual code pegasus and dashboard use. Now they all share the same code.